src/main/java/duke/dukeobject/DukeList.java

```diff
@@ -40,7 +40,9 @@
     protected String viewScope;
     protected String filterCriteria;
 
+    @SuppressWarnings("checkstyle:MemberName")
     private final String DEFAULT_VIEW_SCOPE = "all";
+    @SuppressWarnings("checkstyle:MemberName")
     private final String DEFAULT_SORT_CRITERIA = "time";
```

Comment:
should not need to suppress; either change to static or rename to camel case if these values can vary between different objects of this class (e.g. they are changed in the constructor).